examples/vercel-commerce/lib/shopify/index.ts

```diff
@@ -205,6 +205,10 @@ const reshapeProducts = (products: ShopifyProduct[]) => {
 };
 
 export async function createCart(): Promise<Cart> {
+  if (!process.env.SHOPIFY_STORE_DOMAIN) {
```

Comment:
I think we should avoid modifying the app more than what is needed to run on workers.
If this is to help we the tests, we can skip them when there is an error message.

Reply:
based on this and your other comment I can remove all the modification I did just to enable testing the commerce functionality

Reply:
removed